test/test_mps.py

```diff
@@ -744,6 +746,63 @@ def addDecorator(op, d) -> None:
                          dtypes=MACOS_12_3_XFAILLIST[key]))
         yield op
 
+def mps_ops_error_inputs_modifier(ops):
+    SKIPLIST = {
```

Comment:
SKIPLIST is not the way the to go. We have an expected fail list which should is where unimplemented ops and others are captured.

Reply:
These tests are skipped because the op crashes that cannot be xfailed.

There is a TODO comment below stating the situation.

Reply:
@qqaatw are there are issues describing the crashes? If so, can be referenced in the SKIPLIST comments?
Otherwise, it gives false sense of safety: no open issues in https://github.com/pytorch/pytorch/issues?q=is%3Aopen+is%3Aissue+label%3A%22module%3A+mps%22+label%3A%22module%3A+crash%22 and all tests are green

Reply:
I think currently no, they're newly tested (probably edge) cases. I plan to fix those myself after this pr gets merged and file issues if a crash is not solvable on PyTorch side.

Reply:
Added checks to prevent crashes and removed `SKIPLIST`.